content/docs/traffic-management/transformations/_index.md

```diff
@@ -13,4 +13,4 @@ Mutate and transform requests and responses before forwarding them to the destin
 
 ## Known limitations
 
-**Gateway-level policy in 2.1.x or later**: To apply transformations to all the routes on a Gateway, you must use kgateway version `2.1.0-main` or later. Then, you can select the Gateway in the `targetRefs` field of the TrafficPolicy. Remember that you can configure route-specific transformations by creating another TrafficPolicy with the `targetRefs` field set to the specific HTTPRoute. The HTTPRoute-level policy takes precedence over the Gateway-level policy.
+**Gateway-level policy in 2.1.x or later**: {{< reuse "docs/versions/warn-2-1-only.md" >}} If you use version 2.1 or later, you can select the Gateway in the `targetRefs` field of the TrafficPolicy. Remember that you can configure route-specific transformations by creating another TrafficPolicy with the `targetRefs` field set to the specific HTTPRoute. The HTTPRoute-level policy takes precedence over the Gateway-level policy.
```

Comment:
Probably need to add here that we don't support `HTTPRoute`'s wildcard entries for the different fields of `HTTPCORSFilter` (such as allowOrigins or allowMethods). We only currently support exact string matches on both `HTTPRoute` and `TrafficPolicy`.